core/shared/src/main/scala/zio/logging/LogGroupEquivalence.scala

```diff
@@ -0,0 +1,12 @@
+package zio.logging
+
+final case class LogGroupEquivalence[A](equivalent: (A, A) => Boolean)
```

Comment:
```suggestion
final case class LogGroupEquivalence[-A](equivalent: (A, A) => Boolean)
```

We can easily make this contravariant in `A`.